package.json

```diff
@@ -77,9 +81,5 @@
     "typedoc-plugin-markdown": "~3.11.13",
     "typescript": "~4.5.2",
     "yarn-deduplicate": "^3.1.0"
-  },
-  "resolutions": {
-    "json-schema": "^0.4.0",
-    "validator": "^13.7.0"
```

Comment:
hooray

Reply:
I think these have been moved above in the file and not removed (but maybe they can be removed).

Reply:
ah, nm.

also, might we normalize some of the `~` and `^` usage? or is the distinction between app code (where `~` is correct) and library code (where `^` makes more sense).